src/main/java/io/vertx/ext/mongo/CountOptions.java

```diff
@@ -12,23 +10,28 @@
 
 @DataObject(generateConverter = true)
 public class CountOptions {
-  private static final TimeUnit DEFAULT_MAX_TIME_TIMEUNIT = TimeUnit.MILLISECONDS;
   private JsonObject hint;
   private String hintString;
-  private int limit;
-  private int skip;
-  private long maxTime;
+  private Integer limit;
+  private Integer skip;
+  private Long maxTime;
```

Comment:
Breaking changes

Reply:
Again, these options got introduced in the last PR. Of course, theoretically someone could've used them already. Though it's not very probable. But I respect your change guidelines.

As we rely on these (and other) changes of this PR for https://github.com/NoEnv/vertx-wiremongo (since my last changes were breaking our testing framework) I'd like to integrate them some how. So let me know if I should just document them in the wiki, move them to another PR or both.

Reply:
See my other comment: ok this one time considering the `MongoClient` API must be changed too.